release/package.json

```diff
@@ -615,6 +612,16 @@
           "description": "Controls if type-annotation inlay hints will be displayed for bindings.",
           "type": "boolean"
         },
+        "FSharp.inlineValues.enabled": {
+          "default": false,
+          "description": "Enables rendering all kinds of hints inline with your code. Currently supports pipelineHints, which are like LineLenses that appear along each step of a chain of piped expressions",
+          "type": "boolean"
+        },
+        "FSharp.inlineValues.prefix": {
+          "default": "  // ",
+          "description": "The prefix used when rendering inline values.",
+          "type": "string"
+        },
```

Comment:
Here are the new inlineValues settings to match the config in FSAC - these default to 'off' because the FSAC mechanism isn't fully working for some reason. We maybe should comment them out/remove them entirely for this release until the inlineValues mechanism is proven to be working, but most folks don't mess with settings so it should be safe.